split.Console/Interfaces/IPlayerInterface.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace split.Console.Interfaces
+{
+    /// <summary>
+    /// This interface is a very rough draft. It will eventually be implemented by an 
+    /// object that will encapsulate all player-related behavior. 
+    /// </summary>
+    public interface IPlayerInterface : ICardInterface
```

Comment:
Reasoning for implementing the `ICardInterface`? The player is not a card, instead the player has a _hand_, thus making an `IHaveHand` more logical.